src/utils/index.ts

```diff
@@ -13,3 +16,29 @@ export function extractEmoji(message: string): Array<string> {
         return [];
     }
 }
+
+// Extract the reaction from the message => ReactionManager
+// Get the MessageReaction from the Reaction Manager
+// Iterate and count same reactions.
+// 
+// Return an object with those reactions.
+// *If need be, change emoji.toString() to emoji.toId() to access the ID of the emojis.*
+
+export function extractReaction(message: Message) {
+    const reactionsManager = message.reactions;
+
+    let objreactions: any = {};
+
+    for (var reaction_constructor of reactionsManager.cache) {
+        let ObjReactionMessage = reaction_constructor[1];
+        let emoji = ObjReactionMessage.emoji;
+
+        if (emoji.toString() in objreactions) {
+            objreactions[emoji.toString()] = objreactions[emoji.toString()] + 1;
```

Comment:
Instead of using emoji's name as key instead we can use the id as key.

Reply:
The id might be null so the only other viable option is emoji.identifer.

Reply:
The id being null is very rare case from what I researched. Even discord uses that id for most purposes.